internal/common/consts.go

```diff
@@ -61,6 +63,10 @@ const (
 	// Bump this every time the seed format changes in a backwards incompatible way
 	SeedFormatVersion  = 2
 	SeedFormatOCILabel = "com.openshift.lifecycle-agent.seed_format_version"
+
+	// common file permissions
+	FilePerm600 os.FileMode = 0o600 // For secret files.
```

Comment:
If you're going to create constants for the perms, I would suggest something like RootOnlyRW and RootOnlyRWX. There doesn't seem to be much value in having a constant with the value in the name

Reply:
Additionally, if you're creating constants, use them for all instances. There are multiple places where we already use `0o700` for directories and `0o600` for files. Don't just update the permissions you're changing.

Reply:
Yep makes sense. The intention was have minimal changes so i removed the const to keep it consistent